### ISSUE TITLE
Error: Cannot find module 'fs-pull-blob-store'

### ISSUE THREAD

**Issue by haadcode** [ipfs/js-ipfs#510]
```
Error: Cannot find module 'fs-pull-blob-store'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (<path>/node_modules/ipfs/lib/core/default-repo/node.js:4:13)
```

js-ipfs version 0.16.0

A missing dep?

**Comment by dignifiedquire** [comment]
Are you running in node or in the browser?

**Comment by dignifiedquire** [comment]
The dependency is listed: https://github.com/ipfs/js-ipfs/blob/master/package.json#L70 so maybe forgot to install?

**Comment by diasdavid** [comment]
@haadcode yeah, that sounds like npm has failed you. fresh install should solve your issue.

**Comment by compsocial** [comment]
Fwiw, I got this same error (along with 3 other missing dependencies I resolved manually) during a clean install yesterday.

**Comment by bddap** [comment]
The current release on npm is v0.16.0. In 0.16.0 fs-pull-blob-store is under devDependencies instead of dependencies. [v0.16.0/package.json#L48](https://github.com/ipfs/js-ipfs/blob/a07f16e3b09001d3a88b4b78b9dcab0d5826abb4/package.json#L48)

`npm install ipfs fs-pull-blob-store`

**Comment by diasdavid** [comment]
Apologies about that, it was indeed fixed but never released. Just released 0.17 :)
